src/Database/Schema/MysqlSchema.php

```diff
@@ -28,7 +28,10 @@ class MysqlSchema extends BaseSchema
      */
     public function listTablesSql($config)
     {
-        return ['SHOW TABLES FROM ' . $this->_driver->quoteIdentifier($config['database']), []];
+        return [
+            'SHOW FULL TABLES FROM ' . $this->_driver->quoteIdentifier($config['database']) . ' WHERE Table_Type != ?',
+            ['VIEW']
```

Comment:
Why don't we want views? There have been a bunch of recent issues specifically requesting we better support views in SQLServer.